docs/crio.8.md

```diff
@@ -5,160 +5,128 @@
 crio - OCI Kubernetes Container Runtime daemon
 
 # SYNOPSIS
-**crio**
-[**--apparmor-profile**=[*value*]]
-[**--cgroup-manager**=[*value*]]
-[**--cni-config-dir**=[*value*]]
-[**--cni-plugin-dir**=[*value*]]
-[**--config**=[*value*]]
-[**--conmon**=[*value*]]
-[**--cpu-profile**=[*value*]]
-[**--default-transport**=[*value*]]
-[**--help**|**-h**]
-[**--insecure-registry**=[*value*]]
-[**--listen**=[*value*]]
-[**--log**=[*value*]]
-[**--log-format value**]
-[**--log-level value**]
-[**--pause-command**=[*value*]]
-[**--pause-image**=[*value*]]
-[**--registry**=[*value*]]
-[**--root**=[*value*]]
-[**--runroot**=[*value*]]
-[**--runtime**=[*value*]]
-[**--seccomp-profile**=[*value*]]
-[**--selinux**]
-[**--signature-policy**=[*value*]]
-[**--storage-driver**=[*value*]]
-[**--storage-opt**=[*value*]]
-[**--version**|**-v**]
-
+```
+crio                               \
```

Comment:
This is a nice change over all, but I'm on the fence about the '\' at the end of the line.  How does it look without it?